### ISSUE TITLE
window titles are broken on at least urxvt

### ISSUE THREAD

**Issue by Dieterbe** [ohmyzsh/ohmyzsh#320]
The commit by @nel `Refactor window and tab title in tty` faac3ba8fcad6ad1742f82a7d508a37c0c70d809 breaks window titles.

you can easily verify this:
git checkout faac3ba8fcad6ad1742f82a7d508a37c0c70d809 -> my terminal window titles are just `urxvt` (no matter the pwd).  this is also the case with the latest upstream code.
git checkout faac3ba8fcad6ad1742f82a7d508a37c0c70d809^ -> my terminal window titles are `user@hostname: <pwd>`, as they should be.

on a related note: there should be a bit more documentation on how to work with this new system (assuming it would work).
You see, I used to modify oh-my-zsh like this:
first c82a1c79e358aa8581fed05d3bfeb3bcdbdc15cf
then a bugfix on top of that: b4c4571bea64212b91ce90e1b256aa8717a14718

basically this allowed me to get decent window titles like `user@hostname: <pwd>`, but with a
"shortened" hostname (example: hostname "dieter-ws-a7n8x-arch" gets
printed as "ws")

How can I redo this, with the new title system?
I tried overriding 
`ZSH_THEME_TERM_TAB_TITLE_IDLE` and `ZSH_THEME_TERM_TITLE_IDLE` in my theme
file ( themes/dieter.zsh-theme in the official repository), I also
tried overriding precmd () like so :

```
ZSH_THEME_TERM_TAB_TITLE_IDLE="%15<..<%~%<<" #15 char left truncated PWD
ZSH_THEME_TERM_TITLE_IDLE="%n${hostp} %~"
function precmd {
  title $ZSH_THEME_TERM_TAB_TITLE_IDLE $ZSH_THEME_TERM_TITLE_IDLE
}
```

but since my window title just stays 'urxvt', I can't really test..
